src/microbe_stage/MicrobeStage.cs

```diff
@@ -546,7 +545,8 @@ public void OnReturnFromEditor()
     public void OnFinishTransitioning()
     {
         TransitionFinished = true;
-        TutorialState.SendEvent(TutorialEventType.EnteredMicrobeStage, EventArgs.Empty, this);
+        TutorialState.SendEvent(
+            TutorialEventType.EnteredMicrobeStage, new CallbackEventArgs(HUD.PopupPatchInfo), this);
```

Comment:
This callback won't be set when loading a save. I'll let this slide as it is very unlikely that someone will make a save (or maybe it's impossible to get to the menu yet) while the welcome message is displayed.

Reply:
Would it be sufficient by just adding json property attribute to the action variable in `patchNamePopup`?

Reply:
I don't think so. This is because the callback targets need the specific allow attribute (which is easy to add) but the harder part is that it needs to refer to the HUD object, which is not loaded from JSON, so that would need a special kind of handling to somehow pre-register the HUD instance in the json $id resolver to work.

Reply:
I see.